src/plugins/discover/public/application/main/components/sidebar/discover_index_pattern.test.tsx

```diff
@@ -11,6 +11,7 @@ import { act } from 'react-dom/test-utils';
 import { shallowWithIntl as shallow } from '@kbn/test-jest-helpers';
 import { ShallowWrapper } from 'enzyme';
 import { ChangeIndexPattern } from './change_indexpattern';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { SavedObject } from 'kibana/server';
```

Comment:
@kertal not to myself, this should be `import { SavedObject } from 'kibana/public';`